src/lib/core/option/option.html

```diff
@@ -1,3 +1,5 @@
+<md-pseudo-checkbox class="md-option-pseudo-checkbox" *ngIf="multiple"
```

Comment:
Can potentially cause `Error: The "md-" prefix cannot be used in ng-material v1 compatibility mode.`